testgrid/tgrun/go.mod

```diff
@@ -12,9 +12,9 @@ require (
 	go.uber.org/zap v1.18.1
 	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/api v0.21.3
-	k8s.io/apimachinery v0.21.3
+	k8s.io/apimachinery v0.22.0
 	k8s.io/client-go v12.0.0+incompatible
-	kubevirt.io/client-go v0.43.0
+	kubevirt.io/client-go v0.44.0
```

Comment:
This library is incompatible with k8s.io 1.21+ so I bumped everything to latest patch in 1.20 series.